README.md

```diff
@@ -200,12 +200,13 @@ Given an existing instance of `IStatsDPublisher` called `stats` you can do for e
 ```csharp
 stats.Increment("DoSomething.Attempt");
 var stopWatch = Stopwatch.StartNew();
+var sampleRate = 1;
 var success = DoSomething();
 
 stopWatch.Stop();
 
 var statName = "DoSomething." + success ? "Success" : "Failure";
-stats.Timing(statName, stopWatch.Elapsed);
+stats.Timing(stopWatch.Elapsed, sampleRate, statName);
```

Comment:
Could you remove the sample rate and just swap the parameters so it would be using the convenience extension method? https://github.com/justeattakeaway/JustEat.StatsD/issues/599#issuecomment-1881173538